go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/oam-dev/kubevela
 go 1.19
 
 require (
-	cuelang.org/go v0.5.0-alpha.1
+	cuelang.org/go v0.5.0-beta.2
```

Comment:
shall we upgrade the cue lib here?  \cc @FogDong

Reply:
This upgrade is brought by the upgrade of kubevela/pkg. But in kubevela/kubevela, this upgrade will be overwritten by the `replace` below.